Keas.Core/Domain/Key.cs

```diff
@@ -1,18 +1,13 @@
+using System.Collections.Generic;
 using System.ComponentModel.DataAnnotations;
 
 namespace Keas.Core.Domain
 {
     public class Key : AssetBase {
+        
+        public string Number { get; set; }
 
-        [Required]
-        [StringLength(64)]
-        public string SerialNumber { get; set; }
-
-        public string LockId { get; set; }
-        public Space Space { get; set; }
-        public int? SpaceId { get; set; }
-
-        public KeyAssignment Assignment { get; set; }
-        public int? KeyAssignmentId { get; set; }
+        public List<KeyXSpace> KeyXSpaces { get; set; }
+        public List<Serial> Serials { get; set; }
```

Comment:
@jcubbage @srkirkland should we keep a list of serials here, or should we just have serials point to a key? like spaces and workstations?
right now I'm getting nested stuff in the react, because each serial has its key, and then that key has a list of serials. I'm thinking either we pass in keys to the list, or we remove the list of serials here and group serials by KeyId

Reply:
Pull the list out or mark it not for JSON serialization. I put it there thinking it would help with the queries. But I ended up making serials the base of my queries, so I don't think it got used anywhere.